pyproject.toml

```diff
@@ -13,9 +13,9 @@ exclude = ["example1.py", "example2.py"]
 [tool.poetry.dependencies]
 python = "^3.6.2"
 aiojobs = ">0.2.2"
-fastapi = ">0.55,<0.76"
+fastapi = ">0.55,<0.78"
```

Comment:
Can we drop this constraint `<0.78` now?

Otherwise we will have to tweak the version here every time a new version of fastapi comes out

Reply:
I think it's a good practice to freeze (or at least constraint) dependencies in order to avoid having such breaking changes when dependencies major bumps.

Reply:
ok, let's try